templates/helm/templates/_controller-role-kind-patch.yaml.tpl

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
We only use this file once - to patch the `controller-role` resource. I'm worried that leaving the `_controller-role-kind-patch.yaml` file in the `templates/helm/templates` directory, Helm will try to do a `kubectl apply -f _controller-role-kind-patch.yaml` when we install the template. I think we have 2 choices here:
1. (Probably preferred) Delete the file after we are done patching
2. Add the file name to [`.helmignore`](https://helm.sh/docs/chart_template_guide/helm_ignore_file/) in the directory.